### PR TITLE
Add Elissa to team page

### PR DESCRIPTION
**Pull request by alex-beckett**
## Overview
This PR adds Elissa to the team page.

[Gatsby link](https://celestiaorgteamupdate.gatsbyjs.io/team/)

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: src/datas/team/team-members.js [hunk 1]
```diff
@@ -493,5 +493,18 @@ export const teamMembers = [
             email : '',
             linkedin : '',
          }
+     },{
+        id : 43,
+        name : 'Elissa Greenfields',
```
Comment (jcstein) [inline src/datas/team/team-members.js]:
```suggestion
        name : 'Elissa Greenfield',
```

File: src/datas/team/team-members.js [hunk 2]
```diff
@@ -493,5 +493,18 @@ export const teamMembers = [
...
+        title: 'Head of People Ops',
+        text: 'Former Head of People and Ecosystem Ops at Balancer, plant lover and wellness enthusiast.',
+        image: 'team/ElissaGreenfields.png',
```
Comment (jcstein) [inline src/datas/team/team-members.js]:
```suggestion
        image: 'team/ElissaGreenfield.png',
```

File: src/images/team/ElissaGreenfields.png
Comment (jcstein) [inline src/images/team/ElissaGreenfields.png:1]: the title of this image also needs to be changed

**Review by jcstein — CHANGES_REQUESTED** [review]

**Review by jcstein — APPROVED** [review]

**Comment by alex-beckett** [comment]
closing this, cant get image to show. will do new pr.

**Comment by jcstein** [comment]
your browser may have cached the old image, and is still looking to use the same path to display it, so try a hard refresh of your browser? 

from the interwebs: `Press Ctrl+F5 (If that doesn’t work, try Shift+F5 or Ctrl+Shift+R).`

edit: i see the same thing on the preview link. there's also a chance that the build has old image and path cached

**Pull request closed without merging**